wifiphisher/common/interfaces.py

```diff
@@ -160,6 +181,8 @@ def __init__(self, name, card_obj, mac_address):
         self._has_ap_mode = False
         self._has_monitor_mode = False
         self._is_wireless = False
+        self._is_managed_by_nm = False
```

Comment:
1. `_is_managed_by_nm` : Means whether this card is managed by NetworkManager. I use this property to determine that if the card is deauth or ap and the state is managed, I raise exception. I add this property during property detection since I think it is a card's property should only check once and store it

2. `is_internet_access` : since we can use the wireless card as internet accessing, so this card will also add in `_active` set. when the program exit, `NetworkManager.on_exit` will set the cards in `_active` to `managed mode`, and this will cause the interface down and break the connection of the internet card. So I just add this property to distinguish whether this card is used as internet access and don't set it to managed when `on_exit`